pico/api/src/test/java/io/helidon/pico/api/DefaultPicoConfigTest.java

```diff
@@ -14,12 +14,16 @@
  * limitations under the License.
  */
 
-package io.helidon.pico;
+package io.helidon.pico.api;
 
 import java.util.Map;
 
 import io.helidon.common.config.Config;
 import io.helidon.config.ConfigSources;
+import io.helidon.pico.api.DefaultBootstrap;
```

Comment:
These imports are not required, probably added as part of moving the packages.
You can run `optimize imports` in Idea that only does it for GIT modified files, which helps with this kind of problems

Reply:
I just ran optimize imports now - it didn't find anything additional, but perhaps because It is not comparing against the main branch... will push shortly.

Reply:
pushed